scripts/zkai_native_attention_mlp_rmsnorm_opening_budget_route_gate.py

```
@@ -0,0 +1,841 @@
+            "modeled_delta_vs_two_proof_frontier_after_full_path_opening_removal_bytes",
+            "modeled_frontier_margin_after_full_path_opening_removal_bytes",
+            "strict_margin_after_required_reduction_bytes",
+            "cherry_pick_risk",
+            "policy_sufficient_if_full_path_opening_removed",
+            "promotion_allowed_now",
+            "route_status",
+        },
+        f"route candidate {expected_name}",
+    )
+    if candidate.get("name") != expected_name:
+        raise OpeningBudgetRouteError(f"{expected_name} name drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)